sos/report/plugins/ceph_mds.py

```diff
@@ -0,0 +1,107 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin, UbuntuPlugin
+from socket import gethostname
+import os
+
+
+class CephMDS(Plugin, RedHatPlugin, UbuntuPlugin):
+
+    short_desc = 'CEPH mds'
+
+    plugin_name = 'ceph_mds'
+    profiles = ('storage', 'virt', 'container')
+
+    containers = ('ceph-fs.*',)
+
+    ceph_hostname = gethostname()
+
+    services = (
+        'ceph-mds@%s' % ceph_hostname,
+    )
+
+    def check_enabled(self):
+        # first check services running
+        if super(CephMDS, self).check_enabled():
+            return True
+
+        # also check non empty folders in /var/lib/ceph/mds
+        # in case the mds service is not running for some reason
+
+        mds_folder = '/var/lib/ceph/mds'
+        if os.path.exists(mds_folder) and os.path.isdir(mds_folder):
+            inner_files = os.listdir(mds_folder)
+            for item in inner_files:
+                full_path = mds_folder + "/" + item
+                if(os.path.isdir(full_path)):
+                    # verify the folder has files
+                    if not os.listdir(full_path):
+                        continue
+                    else:
+                        return True
+        return False
```

Comment:
I believe this can all be reduced to the following, for each of the new plugins:

`return True if glob.glob('/var/lib/ceph/mds/*/*') else False`

as `glob.glob` will return `[]` if there is nothing matched at the final location, regardless of if parent directories exist or not.

Reply:
ack, I will test this approach